### ISSUE TITLE
Automobiles Sales Data Analysis

### ISSUE THREAD

**Issue by mariam7084** [abhisheks008/ML-Crate#576]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Automobiles Sales Data Analysis
:red_circle: **Aim** : Perform EDA
:red_circle: **Dataset** : https://www.kaggle.com/datasets/ddosad/auto-sales-data
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Participant ID (If not, then put NA) :
- Approach for this Project :
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
Contributions will start from June 1, 2024. Till then please have some patience.

**Comment by DarkRaiderCB** [comment]
Full name : Sanyog Mishra
    GitHub Profile Link : https://github.com/DarkRaiderCB
    Participant ID: NA
    Approach for this Project : Will perform EDA on dataset provided using techniques like Data cleaning, categorial analysis, finding any outliers, visualisation (correlation matrix, heat maps and more), summary stats., etc. Would utilise feature engineering and use ML algorithms like Linear Regression, Decision Tree, XGBoost, KNN, etc. and will find the best performing model.
    Tools to be used: Pandas, Numpy, Matplotlib, Scikit Learn, XGBoost.
    Resume: https://drive.google.com/file/d/1sDVtq69GJd83t4H1-EOlvHEyQc2oat1k/view?usp=drive_link
    Participant Role: Contributor SSOC Season 3

**Comment by DarkRaiderCB** [comment]
Full name : Sanyog Mishra
GitHub Profile Link : https://github.com/DarkRaiderCB
Participant ID: NA
Approach for this Project : Will perform EDA on dataset provided using techniques like Data cleaning, categorial analysis, finding any outliers, visualisation (correlation matrix, heat maps and more), summary stats., etc. Would utilise feature engineering and use ML algorithms like Linear Regression, Decision Tree, XGBoost, KNN, etc. and will find the best performing model.
Tools to be used: Pandas, Numpy, Matplotlib, Scikit Learn, XGBoost.
Resume: https://drive.google.com/file/d/1sDVtq69GJd83t4H1-EOlvHEyQc2oat1k/view?usp=drive_link
Participant Role: Contributor SSOC Season 3

**Comment by aryan0931** [comment]
Name: Aryan Yadav
github: https://github.com/aryan0931
Participant id:NA
Approach for this project: I will perform Exploratory Data Analysis (EDA) on the provided dataset using techniques such as data cleaning, categorical analysis, outlier detection, and visualization (including correlation matrices, heat maps, and more). This process will involve generating summary statistics and conducting feature engineering to prepare the data for machine learning.
For the machine learning analysis, I will utilize algorithms like Linear Regression, Decision Trees, XGBoost, and K-Nearest Neighbors (KNN) to identify the best-performing model. The tools and libraries used for this analysis will include Pandas, NumPy, Matplotlib, Scikit-Learn, and XGBoost.
Participant role: SSOC

**Comment by abhisheks008** [comment]
> Full name : Sanyog Mishra GitHub Profile Link : https://github.com/DarkRaiderCB Participant ID: NA Approach for this Project : Will perform EDA on dataset provided using techniques like Data cleaning, categorial analysis, finding any outliers, visualisation (correlation matrix, heat maps and more), summary stats., etc. Would utilise feature engineering and use ML algorithms like Linear Regression, Decision Tree, XGBoost, KNN, etc. and will find the best performing model. Tools to be used: Pandas, Numpy, Matplotlib, Scikit Learn, XGBoost. Resume: https://drive.google.com/file/d/1sDVtq69GJd83t4H1-EOlvHEyQc2oat1k/view?usp=drive_link Participant Role: Contributor SSOC Season 3

Implement 5-6 models for this project and compare them based on their accuracy scores.

Assigning this issue to you @DarkRaiderCB

**Comment by DarkRaiderCB** [comment]
> > Full name : Sanyog Mishra GitHub Profile Link : https://github.com/DarkRaiderCB Participant ID: NA Approach for this Project : Will perform EDA on dataset provided using techniques like Data cleaning, categorial analysis, finding any outliers, visualisation (correlation matrix, heat maps and more), summary stats., etc. Would utilise feature engineering and use ML algorithms like Linear Regression, Decision Tree, XGBoost, KNN, etc. and will find the best performing model. Tools to be used: Pandas, Numpy, Matplotlib, Scikit Learn, XGBoost. Resume: https://drive.google.com/file/d/1sDVtq69GJd83t4H1-EOlvHEyQc2oat1k/view?usp=drive_link Participant Role: Contributor SSOC Season 3
> 
> Implement 5-6 models for this project and compare them based on their accuracy scores.
> 
> Assigning this issue to you @DarkRaiderCB

Thanks sir!! @abhisheks008
